netbout-web/src/main/java/com/netbout/rest/MarkdownTxtmark.java

```diff
@@ -110,4 +153,22 @@ private static String makeLineBreak(final String txt) {
         matcher.appendTail(result);
         return result.toString();
     }
+
+    /**
+     * Fix code blocks marked with "```" as they are incorrect processed.
+     * @param txt Text to parse
+     * @return Fixed text with correct code blocks.
+     */
+    private static String fixedCodeBlocks(final String txt) {
+        // @checkstyle LineLengthCheck (1 line)
+        final String eol = "\\u000D\\u000A|[\\u000A\\u000B\\u000C\\u000D\\u0085\\u2028\\u2029]";
+        return txt
+            .replaceAll(
+                String.format("<code>`(%s)?", eol),
```

Comment:
@danff and now there is a need to bring back static variables :) `Pattern.compile` the regex so it will be done only once, not per call to this method